tests/fixtures/dashboard.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import pytz
+from flask import Flask
+
+import flask_monitoringdashboard
+
+
+@pytest.fixture
+def config(username='admin', password='admin', guest_username=None, guest_password=None, colors=None, group_by=None):
+    flask_monitoringdashboard.config.username = username
+    flask_monitoringdashboard.config.password = password
+    flask_monitoringdashboard.config.guest_username = guest_username or 'guest'
+    flask_monitoringdashboard.config.guest_password = guest_password or ['guest_password']
```

Comment:
would it make sense to rename this into `guest_passwords` so it's clear that it's a list?

Reply:
I'm planning on updating user management in another PR, so I won't this now.